changelog/fragments/update-kb-from-9c02d557f01b-to-c158f4fa4207.yaml

```
@@ -0,0 +1,8 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      (go/v2) Fix `create api` when only the controller is created.
+      More info: https://github.com/kubernetes-sigs/kubebuilder/pull/1770
```

The "More info: [link]" string is too verbose. I say we stick to the `([kubebuilder#X](https://github.com/kubernetes-sigs/kubebuilder/pull/))` format, ex.
```suggestion
      (go/v2) Fixed controller imports scaffolded by `create api` when `--resource=false`
      ([kubebuilder#1770](https://github.com/kubernetes-sigs/kubebuilder/pull/1770))
```